stable/cerebro/Chart.yaml

```
@@ -7,7 +7,7 @@ sources:
  - https://github.com/lmenezes/cerebro-docker
  - https://github.com/lmenezes/cerebro
 name: cerebro
-version: 0.5.2
+version: 0.5.3
```

let's update the version to 1.0.0 since this chart is in stable folder :)
```suggestion
version: 1.0.0
```

---

@cpanato: I used the github suggestion commit feature to accept your changes and it has created a DCO problem. Shall I try to revert the commit or would you be able to add your signature?